Spika/Common/Views/UICollectionViewCells/CustomReactionCollectionViewCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CustomReactionCollectionViewCell.swift
+//  Spika
+//
+//  Created by Nikola Barbarić on 12.08.2023..
+//
+
+import Foundation
+import UIKit
+
+class CustomReactionCollectionViewCell: UICollectionViewCell {
+    static let reuseIdentifier = "CustomReactionCollectionViewCell"
```

Comment:
why not override existing reuse identifier?
    override var reuseIdentifier: String? {
        return "Bla"
    }

Reply:
its optional then and making it lot more code